src/test/webview/ui/infoTableComponent.test.ts

```diff
@@ -20,26 +20,26 @@ describe('infoTable', () => {
 
     test('show number of channel (mono)', () => {
         infoTableComponent.showInfo(1, 44100, 1, "s16", "pcm_s16le");
-        expect(document.getElementById('info-table-number_of_channel')?.textContent).toBe('1 (mono)');
+        expect(document.getElementById('info-table-number_of_channel')?.textContent).toBe('1 ch (mono)');
     });
 
     test('show number of channel (stereo)', () => {
         infoTableComponent.showInfo(2, 44100, 1, "s16", "pcm_s16le");
-        expect(document.getElementById('info-table-number_of_channel')?.textContent).toBe('2 (stereo)');
+        expect(document.getElementById('info-table-number_of_channel')?.textContent).toBe('2 ch (stereo)');
     });
 
     test('show sample rate', () => {
         infoTableComponent.showInfo(2, 44100, 1, "s16", "pcm_s16le");
-        expect(document.getElementById('info-table-sample_rate')?.textContent).toBe('44100');
+        expect(document.getElementById('info-table-sample_rate')?.textContent).toBe('44,100 Hz');
     });
 
     test('show file size', () => {
         infoTableComponent.showInfo(2, 44100, 1, "s16", "pcm_s16le");
-        expect(document.getElementById('info-table-file_size')?.textContent).toBe('1 byte');
+        expect(document.getElementById('info-table-file_size')?.textContent).toBe('1 bytes');
     });
 
     test('show duration', () => {
         infoTableComponent.showAdditionalInfo(12.34);
-        expect(document.getElementById('info-table-duration')?.textContent).toBe('12.34s');
+        expect(document.getElementById('info-table-duration')?.textContent).toBe('12.3 s');
```

Comment:
[memo]
If VSCode's language setting is German, etc., "12,3 s" will be displayed on the actual screen, but since Jest's test environment is 'en-US', the test will not fail.
If this project were to test the display in multiple languages, it would need to be modified, but we are not currently planning to do such detailed testing, so we will leave it as is.